Content.Shared/Roles/StartingGearPrototype.cs

```diff
@@ -1,11 +1,18 @@
 using Content.Shared.Preferences;
+using Content.Shared.Preferences.Loadouts;
 using Robust.Shared.Prototypes;
 
 namespace Content.Shared.Roles
 {
     [Prototype("startingGear")]
     public sealed partial class StartingGearPrototype : IPrototype
     {
+        /// <summary>
+        /// The related loadout that this starting gear has.
+        /// </summary>
+        [DataField]
+        public string Loadout = string.Empty;
+
```

Comment:
These prototypes shouldn't be bidirectional, starting gear should be independent of loadouts.

Reply:
ok i just removed all the loadout starting gear changes because it wasn't really even related to the PR, i'll do it in another one